website/pages/en/index.js

```diff
@@ -130,27 +142,29 @@ const StyledGetStartedButton = styled(Button.Anchor)`
 `
 
 const GetStartedButton = ({ href }) => (
-  <StyledGetStartedButton mode="strong" href={href}>Get started</StyledGetStartedButton>
+  <StyledGetStartedButton mode="strong" href={href}>
+    Get started
+  </StyledGetStartedButton>
 )
 
 const BuildingBlocks = props => (
   <BuildingBlocksContainer className="white-section">
     <BlocksCards
       title="aragonOS"
       text={<p>Solidity framework for governance</p>}
-      actionButton={() => <GetStartedButton href={docUrl('aragonos-intro')}/>}
+      actionButton={() => <GetStartedButton href={docUrl('aragonos-intro')} />}
       icon={() => <img src={siteConfig.baseUrl + siteConfig.os} />}
     />
     <BlocksCards
       title="aragonAPI"
       text={<p>Easily interact with your dapp's state</p>}
```

Comment:
`’` can help you avoid the _red-selected-text_.

Reply:
I escaped it with &#39;